django/db/models/fields/related.py

```diff
@@ -2036,6 +2084,12 @@ def set_managed(field, model, cls):
 
 
 class ManyToManyField(RelatedField):
+    # Field flags
+    one_to_many = False
+    one_to_one = False
+    many_to_many = True
+    many_to_one = False
```

Comment:
They could be; however, IMHO there's also value in being explicit at the point where they're actually being used. I'm not too fussed either way, though.